cmd/skopeo/signing.go

```diff
@@ -4,35 +4,67 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
 	"io/ioutil"
 
 	"github.com/containers/image/v5/signature"
-	"github.com/urfave/cli"
+	"github.com/spf13/cobra"
 )
 
 type standaloneSignOptions struct {
 	output string // Output file path
 }
 
-func standaloneSignCmd() cli.Command {
-	opts := standaloneSignOptions{}
-	return cli.Command{
-		Name:      "standalone-sign",
-		Usage:     "Create a signature using local files",
-		ArgsUsage: "MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT",
-		Action:    commandAction(opts.run),
-		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:        "output, o",
-				Usage:       "output the signature to `SIGNATURE`",
-				Destination: &opts.output,
-			},
+func standaloneSignCommand(globalFlagResults *globalOptions) *cobra.Command {
+	var standaloneSignOpts standaloneSignOptions
+	standaloneSignCommand := &cobra.Command{
+		Use:   "standalone-sign [command options] MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT",
+		Short: "Create a signature using local files",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return standaloneSignCmd(cmd, args, standaloneSignOpts)
 		},
+		Example: `skopeo standalone-sign busybox-manifest.json registry.example.com/example/busybox `,
 	}
+
+	standaloneSignCommand.SetUsageTemplate(usageTemplate())
+	standaloneSignCommand.DisableFlagsInUseLine = true
+
+	standaloneSignflags := standaloneSignCommand.Flags()
+	standaloneSignflags.StringVarP(&standaloneSignOpts.output, "output", "o", "", "output the signature to `SIGNATURE`")
+	return standaloneSignCommand
+
 }
 
-func (opts *standaloneSignOptions) run(args []string, stdout io.Writer) error {
+func standaloneVerifyCommand(globalFlagResults *globalOptions) *cobra.Command {
```

Comment:
Please keep the file organized by command, so that definition of flags and their use is close together.